### ISSUE TITLE
Build single page script and criteria for native focus elements

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-website#7515]
### Criteria
Build a single-page test script that can be run without an e2e runner like Nightwatch or Selenium, so we can prove the efficacy of our keyboard e2e tests. This should do static and live checking:

1. We should tally the number of keyboard focusable elements on the page, in order, top to bottom
2. We should evaluate that count is the same as "reachable" tabbed elements. If there are ten items in the DOM that should receive keyboard focus, but only eight that are reachable, this should throw an error.
3. We should evaluate that actual tabbing order is the same as tabbing order defined by `tabindex` and browser
3. We should loop through the tabbable elements on the page and mimic keypress events using a spy or other stand-in, to programmatically verify each tabbable element has been reached. It should be possible to tab to the first element, evaluate it for focus equals true, and repeat until all elements have received keyboard focus and been evaluated for focus.
